pkg/chann/chann.go

```diff
@@ -96,6 +96,7 @@ type Chann[T any] struct {
 	in, out chan T
 	close   chan struct{}
 	cfg     *config
+	q       []T
```

Comment:
```suggestion
    q       []T
	in, out chan T
	close   chan struct{}
	cfg     *config
```
`q` is similar to `in` and `out`.

Reply:
Done